[2021-07-21 06:10:22] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:10:22] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:11:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:12:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:13:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:14:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:15:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:16:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:17:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:18:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:19:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:20:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:21:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:22:19] prodcution.ERROR: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'phone' cannot be null (SQL: insert into `restorants` (`radius`, `name`, `user_id`, `description`, `minimum`, `lat`, `lng`, `address`, `phone`, `active`, `subdomain`, `updated_at`, `created_at`) values ({}, sdgssdds, 45, , 0, 0, 0, , ?, 0, ?, 2021-07-21 06:22:19, 2021-07-21 06:22:19)) {"exception":"[object] (Illuminate\\Database\\QueryException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'phone' cannot be null (SQL: insert into `restorants` (`radius`, `name`, `user_id`, `description`, `minimum`, `lat`, `lng`, `address`, `phone`, `active`, `subdomain`, `updated_at`, `created_at`) values ({}, sdgssdds, 45, , 0, 0, 0, , ?, 0, ?, 2021-07-21 06:22:19, 2021-07-21 06:22:19)) at C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php:692)
[stacktrace]
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(652): Illuminate\\Database\\Connection->runQueryCallback('insert into `re...', Array, Object(Closure))
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(486): Illuminate\\Database\\Connection->run('insert into `re...', Array, Object(Closure))
#2 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(438): Illuminate\\Database\\Connection->statement('insert into `re...', Array)
#3 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Processors\\Processor.php(32): Illuminate\\Database\\Connection->insert('insert into `re...', Array)
#4 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php(2963): Illuminate\\Database\\Query\\Processors\\Processor->processInsertGetId(Object(Illuminate\\Database\\Query\\Builder), 'insert into `re...', Array, 'id')
#5 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php(1617): Illuminate\\Database\\Query\\Builder->insertGetId(Array, 'id')
#6 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1124): Illuminate\\Database\\Eloquent\\Builder->__call('insertGetId', Array)
#7 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1089): Illuminate\\Database\\Eloquent\\Model->insertAndSetId(Object(Illuminate\\Database\\Eloquent\\Builder), Array)
#8 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(930): Illuminate\\Database\\Eloquent\\Model->performInsert(Object(Illuminate\\Database\\Eloquent\\Builder))
#9 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Controllers\\RestorantController.php(609): Illuminate\\Database\\Eloquent\\Model->save()
#10 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Controller.php(54): App\\Http\\Controllers\\RestorantController->storeRegisterRestaurant(Object(Illuminate\\Http\\Request))
#11 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('storeRegisterRe...', Array)
#12 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(254): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\RestorantController), 'storeRegisterRe...')
#13 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(197): Illuminate\\Routing\\Route->runController()
#14 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(695): Illuminate\\Routing\\Route->run()
#15 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#16 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Middleware\\SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#17 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#18 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#19 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#20 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\View\\Middleware\\ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#21 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#22 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Middleware\\Language.php(32): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#23 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): App\\Http\\Middleware\\Language->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#24 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#25 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest(Object(Illuminate\\Http\\Request), Object(Illuminate\\Session\\Store), Object(Closure))
#26 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#27 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#28 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#29 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#30 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#31 C:\\xampp\\htdocs\\lerepass\\packages\\mobidonia\\laravel-dashboard-installer\\src\\Middleware\\ToInstallMiddleware.php(23): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#32 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): dacoto\\LaravelInstaller\\Middleware\\ToInstallMiddleware->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#33 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#34 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(697): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#35 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(672): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#36 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(636): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#37 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(625): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#38 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(166): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#39 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#40 C:\\xampp\\htdocs\\lerepass\\vendor\\barryvdh\\laravel-debugbar\\src\\Middleware\\InjectDebugbar.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#41 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Barryvdh\\Debugbar\\Middleware\\InjectDebugbar->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#42 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#43 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#44 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#45 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#46 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#47 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#48 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#49 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#50 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#51 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#52 C:\\xampp\\htdocs\\lerepass\\vendor\\fruitcake\\laravel-cors\\src\\HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#53 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fruitcake\\Cors\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#54 C:\\xampp\\htdocs\\lerepass\\vendor\\fideloper\\proxy\\src\\TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#55 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#56 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#57 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#58 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#59 C:\\xampp\\htdocs\\lerepass\\public\\index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#60 {main}

[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'phone' cannot be null at C:\\xampp\\htdocs\\lerepass\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDO\\Exception.php:18)
[stacktrace]
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php(114): Doctrine\\DBAL\\Driver\\PDO\\Exception::new(Object(PDOException))
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(485): Doctrine\\DBAL\\Driver\\PDOStatement->execute()
#2 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(685): Illuminate\\Database\\Connection->Illuminate\\Database\\{closure}('insert into `re...', Array)
#3 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(652): Illuminate\\Database\\Connection->runQueryCallback('insert into `re...', Array, Object(Closure))
#4 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(486): Illuminate\\Database\\Connection->run('insert into `re...', Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(438): Illuminate\\Database\\Connection->statement('insert into `re...', Array)
#6 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Processors\\Processor.php(32): Illuminate\\Database\\Connection->insert('insert into `re...', Array)
#7 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php(2963): Illuminate\\Database\\Query\\Processors\\Processor->processInsertGetId(Object(Illuminate\\Database\\Query\\Builder), 'insert into `re...', Array, 'id')
#8 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php(1617): Illuminate\\Database\\Query\\Builder->insertGetId(Array, 'id')
#9 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1124): Illuminate\\Database\\Eloquent\\Builder->__call('insertGetId', Array)
#10 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1089): Illuminate\\Database\\Eloquent\\Model->insertAndSetId(Object(Illuminate\\Database\\Eloquent\\Builder), Array)
#11 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(930): Illuminate\\Database\\Eloquent\\Model->performInsert(Object(Illuminate\\Database\\Eloquent\\Builder))
#12 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Controllers\\RestorantController.php(609): Illuminate\\Database\\Eloquent\\Model->save()
#13 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Controller.php(54): App\\Http\\Controllers\\RestorantController->storeRegisterRestaurant(Object(Illuminate\\Http\\Request))
#14 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('storeRegisterRe...', Array)
#15 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(254): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\RestorantController), 'storeRegisterRe...')
#16 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(197): Illuminate\\Routing\\Route->runController()
#17 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(695): Illuminate\\Routing\\Route->run()
#18 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#19 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Middleware\\SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#20 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#21 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#22 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#23 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\View\\Middleware\\ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#24 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#25 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Middleware\\Language.php(32): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#26 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): App\\Http\\Middleware\\Language->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#27 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#28 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest(Object(Illuminate\\Http\\Request), Object(Illuminate\\Session\\Store), Object(Closure))
#29 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#30 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#31 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#32 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#33 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#34 C:\\xampp\\htdocs\\lerepass\\packages\\mobidonia\\laravel-dashboard-installer\\src\\Middleware\\ToInstallMiddleware.php(23): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#35 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): dacoto\\LaravelInstaller\\Middleware\\ToInstallMiddleware->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#36 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#37 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(697): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#38 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(672): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#39 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(636): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#40 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(625): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#41 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(166): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#42 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#43 C:\\xampp\\htdocs\\lerepass\\vendor\\barryvdh\\laravel-debugbar\\src\\Middleware\\InjectDebugbar.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#44 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Barryvdh\\Debugbar\\Middleware\\InjectDebugbar->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#45 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#46 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#47 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#48 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#49 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#50 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#51 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#52 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#53 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#54 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#55 C:\\xampp\\htdocs\\lerepass\\vendor\\fruitcake\\laravel-cors\\src\\HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#56 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fruitcake\\Cors\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#57 C:\\xampp\\htdocs\\lerepass\\vendor\\fideloper\\proxy\\src\\TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#58 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#59 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#60 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#61 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#62 C:\\xampp\\htdocs\\lerepass\\public\\index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#63 {main}

[previous exception] [object] (PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'phone' cannot be null at C:\\xampp\\htdocs\\lerepass\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:112)
[stacktrace]
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php(112): PDOStatement->execute(NULL)
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(485): Doctrine\\DBAL\\Driver\\PDOStatement->execute()
#2 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(685): Illuminate\\Database\\Connection->Illuminate\\Database\\{closure}('insert into `re...', Array)
#3 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(652): Illuminate\\Database\\Connection->runQueryCallback('insert into `re...', Array, Object(Closure))
#4 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(486): Illuminate\\Database\\Connection->run('insert into `re...', Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php(438): Illuminate\\Database\\Connection->statement('insert into `re...', Array)
#6 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Processors\\Processor.php(32): Illuminate\\Database\\Connection->insert('insert into `re...', Array)
#7 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php(2963): Illuminate\\Database\\Query\\Processors\\Processor->processInsertGetId(Object(Illuminate\\Database\\Query\\Builder), 'insert into `re...', Array, 'id')
#8 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php(1617): Illuminate\\Database\\Query\\Builder->insertGetId(Array, 'id')
#9 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1124): Illuminate\\Database\\Eloquent\\Builder->__call('insertGetId', Array)
#10 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(1089): Illuminate\\Database\\Eloquent\\Model->insertAndSetId(Object(Illuminate\\Database\\Eloquent\\Builder), Array)
#11 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php(930): Illuminate\\Database\\Eloquent\\Model->performInsert(Object(Illuminate\\Database\\Eloquent\\Builder))
#12 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Controllers\\RestorantController.php(609): Illuminate\\Database\\Eloquent\\Model->save()
#13 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Controller.php(54): App\\Http\\Controllers\\RestorantController->storeRegisterRestaurant(Object(Illuminate\\Http\\Request))
#14 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('storeRegisterRe...', Array)
#15 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(254): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\RestorantController), 'storeRegisterRe...')
#16 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(197): Illuminate\\Routing\\Route->runController()
#17 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(695): Illuminate\\Routing\\Route->run()
#18 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#19 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Middleware\\SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#20 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#21 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#22 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#23 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\View\\Middleware\\ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#24 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#25 C:\\xampp\\htdocs\\lerepass\\app\\Http\\Middleware\\Language.php(32): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#26 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): App\\Http\\Middleware\\Language->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#27 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#28 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest(Object(Illuminate\\Http\\Request), Object(Illuminate\\Session\\Store), Object(Closure))
#29 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#30 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#31 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#32 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#33 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#34 C:\\xampp\\htdocs\\lerepass\\packages\\mobidonia\\laravel-dashboard-installer\\src\\Middleware\\ToInstallMiddleware.php(23): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#35 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): dacoto\\LaravelInstaller\\Middleware\\ToInstallMiddleware->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#36 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#37 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(697): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#38 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(672): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))
#39 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(636): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))
#40 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(625): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))
#41 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(166): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))
#42 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))
#43 C:\\xampp\\htdocs\\lerepass\\vendor\\barryvdh\\laravel-debugbar\\src\\Middleware\\InjectDebugbar.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#44 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Barryvdh\\Debugbar\\Middleware\\InjectDebugbar->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#45 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#46 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#47 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#48 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#49 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#50 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#51 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#52 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#53 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#54 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#55 C:\\xampp\\htdocs\\lerepass\\vendor\\fruitcake\\laravel-cors\\src\\HandleCors.php(38): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#56 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fruitcake\\Cors\\HandleCors->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#57 C:\\xampp\\htdocs\\lerepass\\vendor\\fideloper\\proxy\\src\\TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#58 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#59 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#60 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#61 C:\\xampp\\htdocs\\lerepass\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#62 C:\\xampp\\htdocs\\lerepass\\public\\index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#63 {main}
"} 
[2021-07-21 06:22:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:23:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:24:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:25:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:26:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:27:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 06:28:21] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:08:45] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:08:49] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:09:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:10:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:11:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:12:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:13:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:14:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:15:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:16:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:17:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:18:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:19:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:20:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:21:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:22:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:23:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:24:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:25:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:26:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:27:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:28:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:29:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:30:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:31:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:32:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:33:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2021-07-21 18:34:41] prodcution.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in C:\xampp\htdocs\lerepass\app\Models\MyAuth0User.php:8
Stack trace:
#0 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(444): include()
#1 C:\xampp\htdocs\lerepass\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(444): include()
#1 C:\\xampp\\htdocs\\lerepass\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at C:\\xampp\\htdocs\\lerepass\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
